tests/query/parser/test_mql_query.py

```diff
@@ -1374,6 +1374,259 @@
         "metrics",
         id="Select metric with filter for metrics dataset",
     ),
+    pytest.param(
+        'apdex(sum(`d:transactions/duration@millisecond`), 500){dist:["dist1", "dist2"]}',
+        {
+            "entity": "generic_metrics_distributions",
+            "start": "2021-01-01T00:00:00",
+            "end": "2021-01-02T00:00:00",
+            "rollup": {
+                "orderby": "ASC",
+                "granularity": 60,
+                "interval": None,
+                "with_totals": None,
+            },
+            "scope": {
+                "org_ids": [1],
+                "project_ids": [1],
+                "use_case_id": "transactions",
+            },
+            "limit": None,
+            "offset": None,
+            "indexer_mappings": {
+                "d:transactions/duration@millisecond": 123456,
+                "dist": 888,
+            },
+        },
+        Query(
+            QueryEntity(
+                EntityKey.GENERIC_METRICS_DISTRIBUTIONS,
+                get_entity(EntityKey.GENERIC_METRICS_DISTRIBUTIONS).get_data_model(),
+            ),
+            selected_columns=[
+                SelectedExpression(
+                    "aggregate_value",
+                    FunctionCall(
+                        None,
```

Comment:
So this still isn't quite right:

The inner expression of the `SelectedExpression` should also be aliased as `aggregate_value` (which will show up as `_snuba_aggregate_value` here in the test).

That's because if someone references the alias `aggregate_value` elsewhere in the query, right now it won't get the correct expression.